src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/extract_time_fields.js

```diff
@@ -31,8 +36,11 @@ export function extractTimeFields(fields) {
     display: '───',
     fieldName: '',
   };
+  const noTimeFieldLabel = i18n.translate('kbn.management.createIndexPattern.stepTime.noTimeFieldOptionLabel', {
+    defaultMessage: 'I don\'t want to use the Time Filter'
```

Comment:
I would prefer that we use `` or "" for string delimiting if the string contains a ' character.  Easier to read than using escaping.

Reply:
@bmcconaghy Actually, I agree with you that it sometimes maybe better for readabilty (and we use it in react components), but there are points why we did it in such way.
1) We have eslint rule: "Strings must use singlequote"
2) Template string cannot be used, because this restriction is set by tool for extracting messages and verifying them

Reply:
I guess I can live with this as it is.  Is there a possibility of modifying the tool so that template strings would work?

Reply:
Basically there is possibility to do it, but babel creates quite hard AST with template strings, which is difficult to work with, so currently we have decided to use such approach